recipe/conda_build_config.yaml

```
@@ -0,0 +1,2 @@
+MACOSX_SDK_VERSION:        # [osx]
+  - "10.14"                # [osx and x86_64]
```

@vincentmr we were very close! 😅
```suggestion
  - "10.15"                # [osx and x86_64]
```